spec/features/staff/users_can_create_a_fund_spec.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.feature "Users can create a fund" do
+  before do
+    authenticate!(user: user)
+  end
+
+  let(:organisation) { create(:organisation, name: "UKSA") }
+  let(:user) { create(:user, organisations: [organisation]) }
+
+  context "when the user is not logged in" do
+    it "redirects the user to the root path" do
+      page.set_rack_session(userinfo: nil)
+      visit funds_path
+      expect(current_path).to eq(root_path)
+    end
+  end
+
+  scenario "successfully creating a fund" do
+    visit new_fund_path
```

Comment:
I think it is valuable to structure feature tests in such a way that they simulate exactly what the user would do, end-to-end. By starting from the landing page (dashboard_path) and clicking everything in order just as a real user would until they have finished the journey, our test will give us confidence that that feature is available and working.

With the current approach of visiting `new_fund_path` directly in all our tests we have skipped a step that should e responsible for driving us provide navigation.

So long as we have one happy path test that gives us this confidence, I think it is then reasonable to skip the step in subsequent tests for things like validation for efficiency sake.